Dockerfile

```diff
@@ -15,9 +15,8 @@ ENV GOPROXY=${GOPROXY:-https://goproxy.cn}
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
-# Copy the go source
-COPY cmd/core/main.go main.go
-COPY cmd/apiserver/main.go cmd/apiserver/main.go
```

Comment:
Why change them?

Reply:
@Somefive Since I added app/ in cmd/core, I need to copy the entire cmd/core, otherwise the core build image will fail